src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- <h1>Search bar airbnb</h1> -->
+
+    <div class="search-bar-big">
+      <form 
+        data-qa="big"
+        action="https://localhost:3000/api" 
+        method="post" >
```

Comment:
Take a look on checkList
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/230412489-3d43bcec-9cdc-4849-8993-ba39394dfba8.png)

```suggestion
      <form 
        data-qa="big"
        action="https://localhost:3000/api" 
        method="post"
      >
```